Enable columns names by default for CSV in issue #14007 solved

rijaspk opened the pull request:
Enable columns names by default for CSV
Added a export  type check for CSV 
IF true added csv columns 
fixes #14007

Signed-off-by: Rijas Ahammed PK Rijaspk5195@gmail.com

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

Commit:
Enable columns names by default for CSV

Commit:
Improve formatting

Signed-off-by: “Rijas <“Rijas.Ahammed@icfoss.in”>

mauriciofauth commented on libraries/classes/Plugins/Export/ExportCsv.php:
```diff
@@ -113,7 +114,9 @@ protected function setProperties()
-
+        //Enable columns names by default for CSV
+        if($what == 'csv')
+            $GLOBALS['csv_columns'] = 'yes';
```
Hi, @rijaspk. Thanks for your contribution.
Before we can merge your pull request, we would like you to make the following changes.
The body of the `if` structure must be enclosed by braces and there must be a space between the `if` keyword and the parenthesis.

rijaspk replied:
Hi, @mauriciofauth. Required changes are implemented.

Commit:
Follow formatting standard

Enclosed body of if statement within braces
Add space between if keyword and paranthesis

Signed-off-by: “Rijas <“Rijas.Ahammed@icfoss.in”>

mauriciofauth commented:
Merged, thanks for your contribution!

rijaspk commented:
Hi, @mauriciofauth Thanks for reveiwing and accepting my PR
